AuthoringToolTest/PresentationLogic/ModalDialog/ModalDialogInputFieldsFactoryUt.cs

```diff
@@ -9,6 +15,277 @@ namespace AuthoringToolTest.PresentationLogic.ModalDialog;
 
 public class ModalDialogInputFieldsFactoryUt
 {
+    [Test]
+    public void GetCreateLearningElementInputFields_Test()
+    {
+        var systemUnderTest = new ModalDialogInputFieldsFactory();
+
+        for (int contentFileTypeTest = 0; contentFileTypeTest < 8; contentFileTypeTest++)
+        {
```

Comment:
Statt `for` Loop an dieser Stelle, lieber mittels `TestCase` (oder `TestCaseSource`) Parameter in die Testmethode füttern